### DISCUSSION TITLE
Quaternion corrections

### DISCUSSION THREAD

**Discussion by scott198510** [rodralez/NaveGo#112] [Q&A] [answered]
% Quaternion corrections
        % Crassidis. Eq. 7.34 and A.174a.
        antm = [0.0 qua(3) -qua(2); -qua(3) 0.0 qua(1); qua(2) -qua(1) 0.0];

According to Crassidis. Eq. 7.34 and A.174a.

antm = skewm((qua(1) qua(2) qua(3))

Then we get:
 antm = [0.0 -qua(3) qua(2);  qua(3) 0.0 -qua(1); -qua(2) qua(1) 0.0];

There is just a minus sign difference between the code and your paper/book.

@rodralez  Doc. , can you give me some explanations?

**Comment by rodralez** [comment, marked as answer]
Please, checkout the last version of branch `master` . New code for this formula is:

```    
% Crassidis A.174a
Xi = [qua(4)*eye(3) + qua_skew; -qua(1:3)'];
```
